consumer/hack/bind.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+
+#quit if exit status of any cmd is a non-zero value
+set -euo pipefail
+
+SCRIPT_DIR="$(
+    cd "$(dirname "$0")" >/dev/null
+    pwd
+)"
+
+# shellcheck source=consumer/hack/run_test_workload.sh
+source "$SCRIPT_DIR/run_test_workload.sh"
+
+usage() {
+    echo "
+Usage:
+    ${0##*/} [options]
+
+Setup Pipeline Service on a kcp workspace.
+
+Mandatory arguments:
+    -f, --from SERVICE_WORKSPACE
+        Path to the workspace exposing the APIExport.
+
+Optional arguments:
+    -k, --kubeconfig KUBECONFIG
```

Comment:
I think we could think about mandating kubeconfig param too. Because more often than not, the default path for kubeconfig - ~/.kube/config would contain some cluster's kubeconfig and people would have the kcp's kubeconfig stored elsewhere (again, based on the experience with kcp so far, not as a rule). So, it would avoid unnecessary issues if we force the user to provide kubeconfig as manadatory param.

Reply:
I disagree. The user should not have to specify a path every time they want to use the tool. And with kcp only operators would deal with having to log to kcp and clusters. Everyone else should just be logged in kcp.

If we start to do too much hand holding, there's also the fact that a kubeconfig can contain multiple context, and we don't let the user choose which one either, which could also lead to failure.

What I would be open to do, is check that the current kubeconfig/context points to a kcp instance, and return a nice error if that's not the case.

Reply:
I've update the code with the new error message. Feel free to reopen the conversation if that solution is not okay with you.

Reply:
I wasn't too strong about the idea to begin with and I agree with your point on being too strict/constricted with the experience. 
+1 to the error message you've added to check if its pointing to a kcp workspace or not.